tools/bonanza/src/Bonanza/Metrics.hs

```diff
@@ -1,6 +1,8 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE RecordWildCards   #-}
 
+{-# OPTIONS_GHC -Wno-deprecations #-}  -- for Network.BSD
+
```

Comment:
Couldn't you resolve the deprecations?

Reply:
No. `Network.BSD` was moved to `network-bsd`, which is not deprecated but is `network >= 3.0` only. The ecosystem hasn't moved to `network-3.0` yet so we can't use it. Tracking issue: https://github.com/commercialhaskell/stackage/issues/4312

Reply:
My understanding is that those deprecations are kinda "we're going to move it out", not "don't use it because better options are available".